services/src/main/java/org/apache/druid/cli/CliCoordinator.java

```diff
@@ -153,7 +154,8 @@ public class CliCoordinator extends ServerRunnable
 {
   private static final Logger log = new Logger(CliCoordinator.class);
   private static final String AS_OVERLORD_PROPERTY = "druid.coordinator.asOverlord.enabled";
-  private static final String CENTRALIZED_SCHEMA_MANAGEMENT_ENABLED = "druid.centralizedDatasourceSchema.enabled";
+  private static final String CENTRALIZED_DATASOURCE_SCHEMA_ENABLED = "druid.centralizedDatasourceSchema.enabled";
+  private static final String SERVERVIEW_TYPE_PROPERTY = "druid.serverview.type";
```

Comment:
Please push this config to `ServerViewModule` and then reference them from there.